### ISSUE TITLE
RobotAutoloader not load a interface

### ISSUE THREAD

**Issue by Pok4** [nette/robot-loader#29]
Version: Last

I see this error - https://github.com/stripe/stripe-php/issues/1601 in my stripe api.
I write there and i write here :)
The error is because autoloader not load a exception interface. I open the cache and i see only one interface:
```
    'Stripe\\Exception\\OAuth\\ExceptionInterface' => 
    array (
      0 => 'C:\\xampp2\\htdocs\\ext\\pok4\\shop\\stripe\\lib\\Exception\\OAuth\\ExceptionInterface.php',
      1 => 1696003164,
    ),
```

in the bottom of the cache i see this:
`  'C:\\xampp2\\htdocs\\ext\\pok4\\shop\\stripe\\lib\\Exception\\ExceptionInterface.php' => 1696003164,`
maybe this is the problem - this is not load in the above clasess...

The content of this file is
```
<?php

namespace Stripe\Exception;

// TODO: remove this check once we drop support for PHP 5
if (\interface_exists(\Throwable::class, false)) {
    /**
     * The base interface for all Stripe exceptions.
     */
    interface ExceptionInterface extends \Throwable
    {
    }
} else {
    /**
     * The base interface for all Stripe exceptions.
     */
    // phpcs:disable PSR1.Classes.ClassDeclaration.MultipleClasses
    interface ExceptionInterface
    {
    }
    // phpcs:enable
}

```

Do you have any ideas how to load this in robotautoloader ?

**Comment by dg** [comment]
This is by design, RobotLoader only loads classes that are always defined and not just in a condition.

**Comment by Pok4** [comment]
Yes, i get it, but there is a workaround to load it ? Maybe with custom variable ?

**Comment by Pok4** [comment]
When i change the contet of the file to:
```
<?php

namespace Stripe\Exception;


 
interface ExceptionInterface
{
}



```
everything is fine and worked correctly... I'm on PHP 8.1. Maybe the other part with if is not needed...

**Comment by dg** [comment]
I tried adding this [$topLevelOnly](729684f905e5efea384f2497448dfdecc9a7ff6e). Try dev-master pls.

**Comment by Pok4** [comment]
No, not working.. I update with dev master and delete the cache, but in the new cache doesnt find the class.. Only in the bottom:
`   'C:\\xampp2\\htdocs\\ext\\pok4\\shop\\stripe\\lib\\Exception\\ExceptionInterface.php' => 1699278834,`

**Comment by dg** [comment]
Did you delete cache?

**Comment by Pok4** [comment]
Yes - 3 times.

**Comment by dg** [comment]
Then I don't know what the problem is...
